### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Resolves:** #58 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
Hi @blackfalcon, I think auro-badge `WCSS` peer dependency [here](https://github.com/AlaskaAirlines/auro-badge/blob/e173042b3691ac8b297849023c0ecb5733c10ba4/package.json#L30) needs to be updated as well to 5.0.4 for this `ci` git action to pass.
Please correct me if I'm wrong.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
This PR cna not be merged while the tests are failing. Please resolve the testing.

**Comment by fajar-apri-alaska** [comment]
PR is currently `BLOCKED` until below PR have been accepted, merged and shipped.
- https://github.com/AlaskaAirlines/auro-badge/pull/87

**Comment by blackfalcon** [comment]
The auro-badge blocking issue has been resolved. All dependencies have been updated as well as node support.

File: docs/index.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
   <style>
     main {
       width: 80%;
-      margin: var(--auro-size-xl) auto var(--auro-size-xxxl);
-      padding-bottom: var(--auro-size-xxxl);
+      margin: var(--ds-size-400) auto var(--ds-size-800);
```
Comment (blackfalcon) [inline docs/index.html:12]:
```suggestion
      margin: var(--ds-size-400, $ds-size-400) auto var(--ds-size-800, $ds-size-800);
```

Please resolve all missing fallback token references.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (fajar-apri-alaska) [reply docs/index.html:12]: oh wait, this one is html file, SCSS variables couldn't work here, isn't it?

Comment (blackfalcon) [reply docs/index.html:12]: Oops, that's my bad. Knee-jerk response. Nevermind.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**